go-controller/pkg/clustermanager/zone_cluster_controller.go

```diff
@@ -162,6 +182,88 @@ func (zcc *zoneClusterController) syncNodeIds(nodes []interface{}) error {
 	return nil
 }
 
+func (zcc *zoneClusterController) syncZoneJoinSubnet(node *corev1.Node) error {
+	zcc.zoneJoinSubnetAllocator.Lock()
+	defer zcc.zoneJoinSubnetAllocator.Unlock()
+
+	nodeZone := util.GetNodeZone(node)
+	zoneExistingSubnets := zcc.zoneJoinSubnetAllocator.GetZoneSubnets(nodeZone)
+	klog.Infof("Sync Zone join subnets for node [%s] : node zone - [%s] : zone existsing subnets - [%s]", node.Name, nodeZone, zoneExistingSubnets)
```

Comment:
s/existsing/existing
nit: too many white spaces and formating:
```
I0330 10:12:59.899778      41 obj_retry.go:508] Update event received for resource *v1.Node, old object is equal to new: false                                               
I0330 10:12:59.899801      41 zone_cluster_controller.go:145] SURYA node update for ovn-worker                                                                               
I0330 10:12:59.899812      41 zone_cluster_controller.go:228] Sync Zone join subnets for node [ovn-worker] : node zone - [global] : zone existsing subnets - [[100.64.0.0/19]
]                                                                                                                                                                            
I0330 10:12:59.899870      41 zone_cluster_controller.go:275] SURYA I am inside areIPNetsEqual                                                                               
I0330 10:12:59.899916      41 obj_retry.go:508] Update event received for resource *v1.Node, old object is equal to new: false                                               
I0330 10:12:59.900106      41 obj_retry.go:508] Update event received for resource *factory.egressNode, old object is equal to new: false
```

maybe just do:
klog.Infof("Sync join subnets for node %s in zone %s: existing subnets are %s", node.Name, nodeZone, zoneExistingSubnets)